src/huggingface_hub/repocard.py

```diff
@@ -547,6 +549,7 @@ def metadata_eval_result(
     dataset_config: Optional[str] = None,
     dataset_split: Optional[str] = None,
     dataset_revision: Optional[str] = None,
+    metrics_verification_token: Optional[str] = None,
```

Comment:
@Wauplin I added "verification" to the variable name to distinguish from user access tokens - WDYT?

Reply:
Yes makes sense. Thanks for making the change and for the clearer docstring :+1: